lib/commands/kendoui-install.ts

```diff
@@ -0,0 +1,106 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import temp = require("temp");
+import * as path from "path";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUIInstallCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $fs: IFileSystem,
+		private $httpClient: Server.IHttpClient,
+		private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let packages = this.getKendoPackages().wait();
+
+			let downloadUri: string;
+			if(this.$options.latest) {
+				let latestPackage = _.first(packages);
+				let sameDateItems = _.filter(packages, pack => pack.Version === latestPackage.Version);
+				if(sameDateItems.length > 1) {
+					downloadUri = this.selectKendoVersion(sameDateItems).wait();
+				} else {
+					downloadUri = latestPackage.DownloadUrl;
+				}
+			} else {
+				downloadUri = this.selectKendoVersion(packages).wait();
+			}
+
+			let confirm = this.$prompter.confirm(
```

Comment:
I suggest to add support for `--force` option in order to allow using this command in non-interactive terminal, for example:
```TypeScript
let confirm = this.$options.force || this.$prompter.comfirm(...
```